sdlc_xmalab/xma_data_processor.py

```diff
@@ -105,6 +108,82 @@ def list_trials(self):
                                      'analyzed after training into the {path} folder')
         return trials
 
+    def split_rgb(self, trial_path, codec='avc1'):
+        '''Takes a RGB video with different grayscale data written to the R, G, and B channels and splits it back into its component source videos.'''
+        trial_name = os.path.basename(os.path.normpath(trial_path))
+        out_name = trial_name + '_split_'
+
+        try:
+            rgb_video = cv2.VideoCapture(f'{trial_path}/{trial_name}_rgb.avi')
+        except FileNotFoundError as e:
```

Comment:
Similar double-raising of an error as we've discussed previously

Reply:
done